### ISSUE TITLE
(tv.plex.player) Unknown key for dictionary: Keep-Alive

### ISSUE THREAD

**Issue by dlmarston** [plexinc/plex-media-player#380]
Hi team,

I've noticed this error message appearing in my console logs. I'm running OSX 10.11.5 and PMP 1.1.2.359-2b757d45

com.apple.xpc.launchd: (tv.plex.player) Unknown key for dictionary: Keep-Alive

Looking at this website, the KeepAlive tag should not include a minus sign. 

[https://developer.apple.com/library/mac/documentation/MacOSX/Conceptual/BPSystemStartup/Chapters/CreatingLaunchdJobs.html]

The plist seems to want to run /Applications/Plex Media Player.app/Contents/Resources/PMP Helper

But I don't know what this actually does.

**tl;dr** can we remove the typo in Keep-Alive please?

Thanks,
Dave

**Comment by wm4** [comment]
Thanks.

**Comment by dlmarston** [comment]
What does the Helper app do by the way?

**Comment by wm4** [comment]
Make sure only 1 PMP instance runs, process auto updates, collect crash reports, and (not sure if already done or intended to do) listen to input devices for keys that should start PMP. I think.

**Comment by dlmarston** [comment]
ok, so kinda important then :)
